library/js/report_csv.js

```diff
@@ -50,7 +50,12 @@ function export_table_to_csv(html, filename) {
 }
 if (document.querySelector("button") != null) {
     document.querySelector("button").addEventListener("click", function () {
+        let reportname;
+        let rn;
+        reportname = document.getElementById('csv-report').name;
```

Comment:
Keeping the id element the same name throughout the system allows the code to be reused in multiple reports.